webxr-api/device.rs

```diff
@@ -57,4 +58,13 @@ pub trait Device: 'static {
     /// Inputs registered with the device on initialization. More may be added, which
     /// should be communicated through a yet-undecided event mechanism
     fn initial_inputs(&self) -> Vec<InputSource>;
+
+    /// Sets the event handling callback
+    fn set_event_callback(&mut self, callback: Box<dyn EventCallback>);
+
+    /// Whether the device is still connected
+    fn connected(&mut self) -> bool;
```

Comment:
Should this be an event rather than a flag that is polled?

Reply:
This is the communication issue I need to fix